This provides the option to set the indicator for a NFS profile to true.
It may be necessary for users on NFS to use bookmarks/logging.

See: https://bugzilla.mozilla.org/show_bug.cgi?id=1428169

--- firefox-140.1.0/browser/app/profile/firefox.js
+++ firefox-140.1.0/browser/app/profile/firefox.js
@@ -1380,6 +1380,9 @@ pref("browser.quitShortcut.disabled", fa
 // allow META refresh by default
 pref("accessibility.blockautorefresh", false);
 
+// Whether profile is on NFS
+pref("storage.nfs_filesystem", true);
+            
 // Whether history is enabled or not.
 pref("places.history.enabled", true);
 
